### ISSUE TITLE
Is there no other Loss that can be used? I want to do regression tasks

### ISSUE THREAD

**Issue by xlshuhsfa** [Deci-AI/super-gradients#854]

**Comment by ofrimasad** [comment]
Hi @xlshuhsfa .
With SuperGradients you can use any of the native PyTorch Loss functions.
You can find the list here: https://pytorch.org/docs/stable/nn.html#loss-functions
Most common for regression is MSELoss and L1Loss.

To use any of these loss functions, simply instantiate the loss and pass it in the training_params (parameter name is "loss").
